config/locales/en.yml

```diff
@@ -1308,7 +1312,9 @@ en:
       empty: Empty CSV file
       incompatible_type: Incompatible with the selected import type
       invalid_csv_file: 'Invalid CSV file. Error: %{error}'
-      over_rows_processing_limit: contains more than %{count} rows
+      over_rows_processing_limit:
+        one: contains more than %{count} row
+        other: contains more than %{count} rows
```

Comment:
This appears technically correct, however I think it is unnecessary, as `count` here is a constant (currently set to 20000). It may change in the future, but it will always be a large number.